pkg/olog/olog_test.go

```diff
@@ -113,3 +115,37 @@ func TestLogWithAppInfoHook(t *testing.T) {
 		t.Fatalf("unexpected log output (-want +got):\n%s", diff)
 	}
 }
+
+func TestOutputLog(t *testing.T) {
+	// Force JSON handler for valid unmarshaling used in the TestCapturer
+	SetDefaultHandler(JSONHandler)
+
+	logCapture := NewTestCapturer(t)
+	// Initialize test app info
+	app.SetName("ologHooksTest")
+
+	f, err := os.CreateTemp("/tmp", "test_logfile.log")
```

Comment:
For the future, you can use [`t.TempDir()`](https://pkg.go.dev/testing#T.TempDir) to get a test-specific temporary directory that's automatically deleted at the end of the test run.